tests/pw/tests/e2e/productAddons.spec.ts

```
@@ -1,37 +1,47 @@
+    let productName: string;
     let apiUtils: ApiUtils;
 
     // create product addon
+    async function createVendorProductAddon(): Promise<[responseBody, string, string, string, string]> {
     async function createVendorProductAddon(): Promise<[responseBody, string, string, string, string]> {
         const [, categoryId, categoryName] = await apiUtils.createCategory(payloads.createCategoryRandom(), payloads.adminAuth);
         const [responseBody, addonId, addonName, addonFieldTitle] = await apiUtils.createProductAddon({ ...payloads.createGlobalProductAddons(), restrict_to_categories: [categoryId] }, payloads.adminAuth);
+        const [responseBody, addonId, addonName, addonFieldTitle] = await apiUtils.createProductAddon({ ...payloads.createGlobalProductAddons(), restrict_to_categories: [categoryId] }, payloads.adminAuth);
         await dbUtils.updateCell(addonId, VENDOR_ID);
         return [responseBody, addonId, addonName, addonFieldTitle, categoryName];
+        return [responseBody, addonId, addonName, addonFieldTitle, categoryName];
```

Biome

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)